### ISSUE TITLE
[Feature Suggestion] Creative Vortex Tube

### ISSUE THREAD

**Issue by HipHopHuman** [TeamPneumatic/pnc-repressurized#847]
A Creative Vortex Tube, for testing the heat mechanics in creative (like finding the optimal temperature to get the exact Refinery processing speed, or seeing how much temperature is required for a furnace to cook things). Like the Creative Compressor, it can have a GUI where the temperature can be set directly.

Things to consider:

- Should the temperature for both sides be configurable?
- Should it even have sides?
- Perhaps only one side should be configurable, so one can observe the relationship of hot/cold at different temperatures.

I understand this can be done with generating pressure using the Creative Compressor and using the Vortex Tube along with it, but it's a bit of setup to go through just to get a question answered when doing creative testing.

**Comment by desht** [comment]
I'd probably just go with a single creative temperature source rather than a creative vortex tube.  So more like a creative compressed iron block.

But is a 2-block solution (creative compressor & vortex tube) really such a big setup?  Are there any particular use cases which _need_ a 1-block creative heat source?

**Comment by HipHopHuman** [comment]
> I'd probably just go with a single creative temperature source rather than a creative vortex tube. So more like a creative compressed iron block.

That's fine - I suggested the Vortex Tube because it's the one block that is mostly associated with the temperature mechanic, but Creative Compressed Iron works too.

> But is a 2-block solution (creative compressor & vortex tube) really such a big setup? Are there any particular use cases which _need_ a 1-block creative heat source?

It's not the *size* of the setup that is the issue, it's the guesswork involved in setting the pressure right to get the desired temperature. It's a bit of a pain point in creative testing to have to eyeball the temperature and adjust the compressor accordingly - to a person doing this for the first time, it will take a few attempts. 

Normally I've never had this issue, but when demonstrating the Create + PNC:R heating mechanics in the showcase channel on Discord, it took me around 30-45 minutes to find the right pressure level to get the temperature precise enough that it would extract the iron ore before smelting it, while progressing the smelt far enough that the rotation of the furnace engine was constant (if it was too short, the rotation would intermittently pause). It may not seem like a big deal but it was multiple attempts of trial and error.

With a creative temperature item, finding the optimal temperature would have taken just a few seconds.

If you're not into adding this suggestion, I don't mind, because now that I have the experience, I can live without it. I've just observed one particular pain point of the mod for new players - albeit a small one - that could potentially be improved to reduce the barrier of entry (in making testing the mod's mechanics a little easier)
